tests/integration/test_local_client.py

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+import pytest
+
+import re
+
+import numpy as np
+import pandas as pd
+
+from antares import create_study_local
+from antares.exceptions.exceptions import (
+    AreaCreationError,
+    InvalidNameError,
+    LinkCreationError,
+)
+from antares.model.area import AdequacyPatchMode, Area, AreaProperties, AreaUi
+from antares.model.binding_constraint import BindingConstraintProperties, ClusterData, ConstraintTerm, LinkData
+from antares.model.commons import FilterOption
+from antares.model.link import Link, LinkProperties, LinkUi
+from antares.model.renewable import RenewableClusterGroup, RenewableClusterProperties
+from antares.model.settings.general import GeneralParametersLocal, Mode
+from antares.model.settings.playlist_parameters import PlaylistParameters
+from antares.model.settings.study_settings import StudySettingsLocal
+from antares.model.st_storage import STStorageGroup, STStorageProperties
+from antares.model.study import Study
+from antares.model.thermal import ThermalCluster, ThermalClusterGroup, ThermalClusterProperties
+from antares.tools.ini_tool import IniFile, IniFileTypes
+from antares.tools.time_series_tool import TimeSeriesFileType
+
+
+class TestLocalClient:
+    def test_local_study(self, tmp_path, other_area):
```

Comment:
Add a msg at the beginning of the test to explain what it's gonna perform

Reply:
Added.